### DISCUSSION TITLE
Whether macOS Notarization would affect updater signature?

### DISCUSSION THREAD

**Discussion by MikeChongCan** [tauri-apps/tauri#7703] [Q&A] [answered]
The current workflow is like:
1. `pnpm tauri build` -> build the app and compute the signature
2. zip the `app.tar.gz` to `app.tar.gz.zip` and submit it for notarization

After (2), should we update the signature? If so, how to manually compute the signature?

**Comment by FabianLars** [comment, marked as answer]
I don't think it has an effect on it, but in general you can always use the `pnpm tauri signer sign` command (check its `--help` output for instructions) on the files you want to get a new signature for.

**Reply by MikeChongCan** [reply]
Got it! Thank you!

**Reply by vlev1n** [reply]
Just a reminder for other people. `pnpm tauri signer -h` and `pnpm tauri signer sign -h` are totally different things. Usually, the latter one is what you want.
